WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -366,6 +382,28 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForWritingSettingsAt
     return nil;
 }
 
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForDeviceSettingsAtRow:(NSInteger)row
+{
+    switch (row) {
+        case (SiteSettingsDeviceGeotagging):{
+            return self.geotaggingCell;
+        }
+            break;
+        case (SiteSettingsDeviceDefaultCategory):{
+            [self configureDefaultCategoryCell];
+            return self.defaultCategoryCell;
+        }
+            break;
+        case (SiteSettingsDeviceDefaultPostFormat):{
+            [self configureDefaultPostFormatCell];
+            return self.defaultPostFormatCell;
+        }
+            break;
```

Comment:
The style consistency of the switch cases and breaks in this file are all over the place. I'm going to leave this alone for now as it would require touching a lot more code than the context of this PR.